src/VolleyManagement.WebClient/spa/app/Components/Login/login.component.html

```diff
@@ -0,0 +1,7 @@
+<div>
```

Comment:
What is the point of this div? I mean? it seems you do not apply any specific styles to it.

Reply:
I used it as a 'container' for content.